packages/block-library/src/verse/block.json

```
@@ -10,12 +10,24 @@
 			"default": "",
 			"__unstablePreserveWhiteSpace": true
 		},
+		"style": {
+			"default": {
+				"spacing": {
+					"padding": {
+						"default": 20
```

hey, this needs to be (see [example in docs](https://developer.wordpress.org/block-editor/developers/block-api/block-supports/#spacing)):

```suggestion
						"top": 20,
						"right": 20,
						"bottom": 20,
						"left": 20
```